### DISCUSSION TITLE
None of the modules are working for me  of LSposed.

### DISCUSSION THREAD

**Discussion by Manishwq** [LSPosed/LSPosed#2558] [General]
Anybody help me out please. I have tried many times but none of the modules of LSposed are working even LSposed gyzisk is activated. What to do. The modules are successfully installed but not working. I have tried classic power menu, adaway for YouTube and more but none of them working.

**Comment by mywalkb** [comment]
without logs is impossible to help you.
Have you set the rigth scope for each module?
No version of android, no version of LSPosed used. too low informations.

**Reply by Manishwq** [reply]
I am using redmi 9 activ. Android 11, MIUI 12.5
Magisk 25.0, because latest magisk cause bootloop, and LSposed 1.8.3. Using gyzisk version of LSposed and zygisk enabled in magisk. Looks everything correct but don't know what's missing. Tried all possibilities from my side but nothing happened. Kindly suggest something.
![Screenshot_2023-06-09-21-52-00-943_com android shell](https://github.com/LSPosed/LSPosed/assets/136048189/dd6f53bc-34bb-4990-a239-ed97ecd9d623)

**Reply by mywalkb** [reply]
LSPosed is corretly installed and working also is not the latest version.
Which modules have you installed? I see 2 modules.
Try to post logs, you can save from manager.

**Reply by Manishwq** [reply]
[LSPosed_2023-06-10T00_32_18.791.zip](https://github.com/LSPosed/LSPosed/files/11709363/LSPosed_2023-06-10T00_32_18.791.zip)

Also tried with LSposed latest version but didn't work.

![Screenshot_2023-06-10-00-38-28-184_com android shell](https://github.com/LSPosed/LSPosed/assets/136048189/cf1db11d-3467-4434-840d-202437e540e6)

**Reply by mywalkb** [reply]
For module Youtube AdAway the issue is on the version of youtube app:

```
[ 2023-06-09T23:32:56.235    10201: 15711: 15711 I/LSPosed-Bridge  ] Loading module ma.wanam.youtubeadaway from /data/app/~~CtsbGyMcZ6T_iLhwc2OkHg==/ma.wanam.youtubeadaway-hd7D10FX7bquQea97ZSJYA==/base.apk
[ 2023-06-09T23:32:56.392    10201: 15711: 15711 I/LSPosed-Bridge  ]   Loading class ma.wanam.youtubeadaway.Xposed
[ 2023-06-09T23:32:56.478    10201: 15711: 15711 I/LSPosed-Bridge  ] Hooking YouTube version: com.google.android.youtube 1527766464
[ 2023-06-09T23:32:59.011    10201: 15711: 15746 I/LSPosed-Bridge  ] Found emptyComponent class: dme.a
[ 2023-06-09T23:33:02.628    10201: 15711: 15746 I/LSPosed-Bridge  ] Found ad cards class: nca.c
[ 2023-06-09T23:33:02.632    10201: 15711: 15746 I/LSPosed-Bridge  ] Ad cards hooks applied in 6 seconds!
>>>>>>>> [ 2023-06-09T23:33:04.802    10201: 15711: 15711 I/LSPosed-Bridge  ] YouTube AdAway: brute force failed! <<<<<<<<<
```
Hooks are working but the module is incompatible with the version of your youtube app.
In the log you can find this "Found emptyComponent class" which means the hook code of module is executed without issue.
**the official YouTube app Tested against YouTube 17.40.40**
I think your version 1527766464 17.07.35 is too old.
Try to install that exact version.

For Classic Power Menu I'm still analyzing, but I think there is a similar issue.
LSPosed is working without issue.

**Reply by Manishwq** [reply]
Ok thanks buddy. I will install that version

**Reply by mywalkb** [reply]
ClassicPowerMenu has no info log only errors and no errors is logged so should works.
ClassicPowerMenu has 2 ways for works xposed and Accessibility Service.
Xposed is not required but should react quicker than Accessibility Service.

**Reply by Manishwq** [reply]
Done with accessibility. But not worked.

I tried YouTube adaway with the said version. Ad blocker perfectly works. Many thanks for you man. But background play is not working can you help me please.

**Reply by mywalkb** [reply]
> Done with accessibility. But not worked.

you should open an issue on [this](https://github.com/KieronQuinn/ClassicPowerMenu) repository, the developer can help you better of me.

> I tried YouTube adaway with the said version. Ad blocker perfectly works. Many thanks for you man. But background play is not working can you help me please.

also in this case I suggest you to open an issue on [this](https://github.com/wanam/YouTubeAdAway) repository. There is a similar [issue](https://github.com/wanam/YouTubeAdAway/issues/115), but it's closed.
